source/Minable.h

```diff
@@ -38,6 +38,21 @@ class Visual;
 // Class representing an asteroid or other minable object that orbits in an
 // ellipse around the system center.
 class Minable : public Body {
+public:
+	class Payload {
+	public:
+		Payload(const DataNode &node);
+
+		const Outfit *outfit;
+		// The maximum number of outfits that this payload can drop.
+		int dropSize = 1;
```

Comment:
The variable name could be clearer I think
maxDrop mb?
dropLimit?
maxDropAmount sounds too long for sure

Reply:
Changed to maxDrops.